mkdocs.yml

```diff
@@ -57,6 +57,13 @@ pages:
   - Troubleshooting: extensions/troubleshooting.md # page-tree = DONE
   - Advanced patterns: extensions/advanced.md
 - Framework Reference:
```

Comment:
Rebased (to address merge-conflits) and updated to use this structure.

Note: I also renamed "Modifications" to "Changesets". This matches terminology that is used in the code, and it feels a bit more precise.